app/assets/stylesheets/db/common/base.scss

```diff
@@ -0,0 +1,39 @@
+body {
+  padding-top: 80px;
+}
+
+nav.navbar {
+  height: 60px;
+  line-height: 60px;
+
+  .navbar-header .navbar-brand {
+    height: 60px;
+    line-height: 60px;
+    padding: 0 15px;
+  }
+
+  .navbar-collapse {
+    .navbar-nav a {
+      line-height: 60px;
+      padding: 0 15px;
+    }
+
+    form.navbar-form {
+      height: 60px;
+      line-height: 56px;
+      margin: 0;
+
+      input.form-control {
+        width: 300px;
+      }
+    }
+  }
+}
+
+.edit-type-tabs {
+  margin-bottom: 30px;
+}
+
+ul.resource-nav-pills {
+  margin: 10px 0 20px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `10px 0 20px` instead of `10px 0 20px 0`